docs/latin.rst

```diff
@@ -721,6 +721,44 @@ The StringUtils module contains utility methods for processing scansion and text
    Out[2]: 'I m ok  Oh              Fine'
 
 
+Semantics
+=========
+The Semantics module allows for the lookup of Latin lemmata, synonyms, and translations into Greek. Lemma, synonym, and translation dictionaries are drawn from the open-source `Tesserae Project<http://github.com/tesserae/tesserae>`
+
+.. tip:: When lemmatizing ambiguous forms, the Semantics module is designed to return all possibilities. A probability distribution is included with the list of results, but as of June 8, 2018 the total probability is evenly distributed over all possibilities. Future updates will include a more intelligent system for determining the most likely lemma, synonym, or translation._.
+
+The Lemmata class includes two relevant methods: lookup() takes a list of tokens standardized for spelling and returns a complex object which includes a probability distribution; isolate() takes the object returned by lookup() and discards everything but the lemmata.
+
+.. code-block:: python
+
+   In [1]: from cltk.semantics.latin.lookup import Lemmata
+
+   In [2]: lemmatizer = Lemmata(dictionary = 'lemmata', language = 'latin')
+
+   In [3]: tokens = ['ceterum', 'antequam', 'destinata', 'componam']
+
+   In [4]: lemmas = lemmatizer.lookup(tokens)
+   Out[4]:
+   [('ceterum', [('ceterus', 1.0)]), ('antequam', [('antequam', 1.0)]), ('destinata', [('destinatus', 0.25), ('destinatum', 0.25), ('destinata', 0.25), ('destino', 0.25)]), ('componam', [('compono', 1.0)])]
+
+   In [5]: justlemmas = lemmatizer.isolate(lemmas)
+   Out[5]:['ceterus', 'antequam', 'destinatus', 'destinatum', 'destinata', 'destino', 'compono']
+
+The Synonym class can be initialized to lookup either synonyms or translations. It expects a list of lemmata, not inflected forms. Only successful 'lookups' will return results.
+
+.. code-block:: python
+
+   In [1]: from cltk.semantics.latin.lookup import Synonyms
+
+   In [2]: translator = Synonyms(dictionary = 'translations', language = 'latin')
```

Comment:
I know this is a work-in-progress, so don't worry about it for now … however: We will want to document clearly the possible params for `dictionary` for `Synonyms` (same for `Lemmata`, above).